style.css

```diff
@@ -240,14 +323,46 @@ input[type="submit"]:hover {
   margin-bottom: 24px;
 }
 
+@media screen and (min-width: 600px) {
+  .about .description {
+    max-width: 60%;
+  }
+
+  .about .crane.right {
+    transform: scale(3);
+    right: 0;
+    top: -50px;
+  }
+
+  .about .crane.left {
+    left: 0;
+    top: 100px;
+    transform: scale(3);
+  }
+}
+
 .about .card-container {
   margin-top: 59px;
 }
 
+@media screen and (min-width: 800px) {
```

Comment:
- [ ] Please let us use the range of 992px - 1100px for the min-width in the `@media queries`, because this is actually the size of the desktop screen.

Reply:
Thank you. I'll adjust my media queries to match this.